.circleci/config.yml

```diff
@@ -498,15 +498,22 @@ jobs:
           name: Tag images
           command: |
             docker images fundocker/marsha
-            DOCKER_TAG=$(echo ${CIRCLE_TAG} | sed 's/^v//')
-            echo "DOCKER_TAG: ${DOCKER_TAG} (from Git tag: ${CIRCLE_TAG})"
-            docker tag marsha:${CIRCLE_SHA1} fundocker/marsha:latest
+            DOCKER_TAG=$([[ -z "$CIRCLE_TAG" ]] && echo $CIRCLE_BRANCH || echo ${CIRCLE_TAG} | sed 's/^v//')
+            RELEASE_TYPE=$([[ -z "$CIRCLE_TAG" ]] && echo "branch" || echo "tag ")
+            # Display either:
+            # - DOCKER_TAG: master (Git branch)
+            # or
+            # - DOCKER_TAG: 1.1.2 (Git tag v1.1.2)
+            echo "DOCKER_TAG: ${DOCKER_TAG} (Git ${RELEASE_TYPE}${CIRCLE_TAG})"
             docker tag marsha:${CIRCLE_SHA1} fundocker/marsha:${DOCKER_TAG}
             docker tag marsha:${CIRCLE_SHA1}-dev fundocker/marsha:${DOCKER_TAG}-dev
-            docker tag marsha:${CIRCLE_SHA1}-alpine fundocker/marsha:alpine
             docker tag marsha:${CIRCLE_SHA1}-alpine fundocker/marsha:${DOCKER_TAG}-alpine
             docker tag marsha:${CIRCLE_SHA1}-alpine-dev fundocker/marsha:${DOCKER_TAG}-alpine-dev
             docker images "fundocker/marsha:${DOCKER_TAG}*"
+            if [[ ! -z "$CIRCLE_TAG" ]]; then
+              docker tag marsha:${CIRCLE_SHA1} fundocker/marsha:latest
```

Comment:
we want to keep `latest` as the latest tagged image.
In fact that we want is if it's master branch then tag images with tag `master` and `master-alpine`. Else keep the tag we have.

Reply:
Do not take care of my comment, I didn't read correctly your if condition.